easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '6.0.1'
+
+homepage = 'http://gnuplot.sourceforge.net'
+description = """Portable interactive, function plotting utility"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = [('https://sourceforge.net/projects/%(name)s/files/%(name)s/%(version)s', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['e85a660c1a2a1808ff24f7e69981ffcbac66a45c9dcf711b65610b26ea71379a']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '2.0.3'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('ncurses', '6.4'),
+    ('cairo', '1.18.0'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('libpng', '1.6.40'),
+    ('libgd', '2.3.3'),
+    ('Pango', '1.51.0'),
+    ('libcerf', '2.4'),
+    ('X11', '20231019'),
+    ('Qt6', '6.6.3'),
+    ('Lua', '5.4.6'),
+]
+
+preconfigopts = 'autoreconf && '
+
+# make sure that right Lua library is used (bypassing pkg-config)
+preconfigopts += 'export LUA_CFLAGS="-I$EBROOTLUA/include" && export LUA_LIBS="$EBROOTLUA/lib/liblua.a" && '
+
+# fix undefined reference to symbol 'libiconv_open'
+preconfigopts += ' export LDFLAGS="-Wl,--copy-dt-needed-entries" && '
+
+configopts = '--with-qt=qt6 --without-latex --disable-wxwidgets'
+
+buildopts = 'LDFLAGS="$LDFLAGS -Wl,--copy-dt-needed-entries" '
```

Comment:
I don't think you need this one, only the preconfigopts one is used in the version for 13.3.0